applications/nrf5340_audio/src/main.c

```diff
@@ -215,18 +178,9 @@ int main(void)
 
 #if defined(CONFIG_AUDIO_DFU_ENABLE)
 	/* Check DFU BTN before initialize BLE */
-	dfu_entry_check((void *)ble_core_init);
+	dfu_entry_check((void *)bt_mgmt_init);
```

Comment:
@rick1082 Please have a good look at this one.
@koffes: Will run extra tests on DFU.

Reply:
There will be an error when build the code with DFU feature enabled:
/home/rick/ncs_audio/nrf/applications/nrf5340_audio/src/main.c:181:33: error: 'bt_mgmt_init' undeclared (first use in this function); did you mean 'bt_mgmt_chan'?
  181 |         dfu_entry_check((void *)bt_mgmt_init);
      |                                 ^~~~~~~~~~~~
      |                                 bt_mgmt_chan

The following line should be added:
#include "bt_mgmt.h"